public/js/index.js

```diff
@@ -1786,6 +1787,14 @@ function initVueComponents(){
     })
 }
 
+function initCtrlEnterSubmit() {
+    $(".ctrlenter").keydown(function(e) {
```

Comment:
I was searching for better solution than using class. We can use custom attribute. e.g. "submit" (or "keysubmit", "autosubmit", 'submit="ctrl+enter"', ...) and do simple jq select like `$("textarea[autosubmit]")` or `$("[autosubmit]")` or `$["[submit='ctrl+enter']"]`.

Reply:
@lafriks What do you think?

Reply:
why not use the custom `data-` attributes? https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

Reply:
I am not (pure/jq) js expert, but is not purpose of `data-` attributes for storing data? We need only element selector. Using `data-` would work too, but it has other meaning and purpose for me.

Reply:
E.g. boostrap uses `data-` attributes for their javascript handlers intensively

Reply:
@Morlinest What are the advantages of custom attributes?

Reply:
@daviian yes but mostly for data. In most cases bootstrap uses classes to attach events to elements

Reply:
@JonasFranzDEV It's not about advantage. It's about expresivness meaning. Personaly if I find something in `class` attribute, I would search for definition in css or expect some kind of UI changes done by js (simple rule: "class" = how element looks). With this solution (custom attribute, same as `data-*` - just for another purpose), you don't need to mix styles with events. It's all only my opinion and suggestion.

Reply:
@Morlinest How good is browser integration for custom element attributes? IE < 10 as well? Does gitea support IE < 10?

Reply:
@daviian gitea supports only browsers that semantic UI does so no official support for IE < 10

Reply:
Ok so the only downside of a custom attribute is that validators don't attest that it's valid html.

Reply:
We are also using Vue (that means custom elements and attributes), so I don't see any problem here. Any js framework use some kind of custom elements/attributes, I think it's very common in web.

Reply:
I don't mind for any of options mentioned here but for sake of consistency I would leave class as it is what we use in other places in code

Reply:
If you want to better follow html standards for pure js/jquery, we can use mentioned `data-submit="ctrl+enter"`.

Reply:
@Morlinest Now that you mention Vue I see that html validity is off the table ;-)

Reply:
@Morlinest @daviian So can we agree on solution so that either @JonasFranzDEV can fix this or we can merge it as is

Reply:
@lafriks For me it is ok as it is. The class could be a little more descriptive, at first glance nobody would know that it is for form submission. Something like `can-ctrlenter-submit`

But I don't want to enforce my opinion.

Reply:
Or just take inspiration from github and use "js-quick-submit" class name.

Reply:
@JonasFranzDEV please change class name to @Morlinest suggested

Reply:
@lafriks Done